Failed to Load Native Binding in Android Arm64

I am using Android Arm64 with Termux as Terminal want to create a new project using Nuxtjs latest version. However, there is an error Failed to Load Native Binding from oxc-parser during the installation process.

Here is the error log

``` 
[6:41:55 AM]  ERROR  Failed to load native binding
    at Object.<anonymous> (node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js:380:11)
    at Module._compile (node:internal/modules/cjs/loader:1734:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1899:10)
    at Module.load (node:internal/modules/cjs/loader:1469:32)
    at Module._load (node:internal/modules/cjs/loader:1286:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1491:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/index.js:2:18)
  [cause]: [ { Error: Cannot find module './parser.android-arm64.node'
  Require stack:
/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js
      at Module._resolveFilename (node:internal/modules/cjs/loader:1405:15)
      at defaultResolveImpl (node:internal/modules/cjs/loader:1061:19)
      at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1066:22)
      at Module._load (node:internal/modules/cjs/loader:1215:37)
      at TracingChannel.traceSync (node:diagnostics_channel:322:14)
      at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
      at Module.require (node:internal/modules/cjs/loader:1491:12)
      at require (node:internal/modules/helpers:135:16)
      at requireNative (/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js:76:16)
      at Object.<anonymous> (/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js:345:17)
    code: 'MODULE_NOT_FOUND',
    requireStack:
     [ '/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js' ] },
  { Error: Cannot find module '@oxc-parser/binding-android-arm64'
  Require stack:
/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js
      at Module._resolveFilename (node:internal/modules/cjs/loader:1405:15)
      at defaultResolveImpl (node:internal/modules/cjs/loader:1061:19)
      at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1066:22)
      at Module._load (node:internal/modules/cjs/loader:1215:37)
      at TracingChannel.traceSync (node:diagnostics_channel:322:14)
      at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
      at Module.require (node:internal/modules/cjs/loader:1491:12)
      at require (node:internal/modules/helpers:135:16)
      at requireNative (/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js:81:16)
      at Object.<anonymous> (/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js:345:17)
    code: 'MODULE_NOT_FOUND',
    requireStack:
     [ '/data/data/com.termux/files/home/testpnpm/node_modules/.pnpm/oxc-parser@0.72.3/node_modules/oxc-parser/bindings.js' ] } ]
[6:41:55 AM]  ERROR  Failed to load native binding ELIFECYCLE  Command failed with exit code 1.
[nuxi 6:41:55 AM]  ERROR  Error: corepack pnpm install failed.
```

--- a/.github/workflows/reusable_release_napi.yml
+++ b/.github/workflows/reusable_release_napi.yml
@@ -113,6 +113,10 @@ jobs:
               pnpm build --target s390x-unknown-linux-gnu --use-napi-cross
 
           - os: ubuntu-latest
+            target: aarch64-linux-android
+            build: pnpm build --target aarch64-linux-android
+
+          - os: ubuntu-latest
             target: riscv64gc-unknown-linux-gnu
             build: |
               sudo apt-get update
@@ -162,7 +166,7 @@ jobs:
           RUSTUP_IO_THREADS: 1
         with:
           operating_system: freebsd
-          version: "14.0"
+          version: "14.2"
           memory: 8G
           cpu_count: 3
           environment_variables: "DEBUG RUSTUP_IO_THREADS"

--- a/napi/minify/package.json
+++ b/napi/minify/package.json
@@ -60,6 +60,7 @@
       "riscv64gc-unknown-linux-gnu",
       "x86_64-apple-darwin",
       "aarch64-apple-darwin",
+      "aarch64-linux-android",
       "wasm32-wasip1-threads"
     ],
     "wasm": {

--- a/napi/parser/package.json
+++ b/napi/parser/package.json
@@ -86,6 +86,7 @@
       "riscv64gc-unknown-linux-gnu",
       "x86_64-apple-darwin",
       "aarch64-apple-darwin",
+      "aarch64-linux-android",
       "wasm32-wasip1-threads"
     ],
     "wasm": {

--- a/napi/transform/package.json
+++ b/napi/transform/package.json
@@ -60,6 +60,7 @@
       "riscv64gc-unknown-linux-gnu",
       "x86_64-apple-darwin",
       "aarch64-apple-darwin",
+      "aarch64-linux-android",
       "wasm32-wasip1-threads"
     ],
     "wasm": {

